### DISCUSSION TITLE
Apple iPads

### DISCUSSION THREAD

**Discussion by M-Sport** [community/community#132266] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi all
Does anyone know of a way to update OS on ipads, there are too many ipads out their going to waste

**Comment by ebndev** [comment]
Hi @M-Sport,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as an Apple forum. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by M-Sport** [reply]
Hi, I’m new to Git Hub, how do I put this post in the Apple forum?

**Reply by ebndev** [reply]
Hi! I meant a forum that's not on GitHub Discussions, such as Reddit or another place where you might find more Apple users 😄
